### PR TITLE
Update 60-evdev.hwdb

### PR DESCRIPTION
**Pull request by davijosw**
By the libinput measure touchpad-size tool, I identify that the touchpad's precision needs to be improved. If some adjust is in need let me know,

### CODE REVIEW

**Comment by davijosw** [comment]
https://github.com/systemd/systemd/issues/23120

File: hwdb.d/60-evdev.hwdb [hunk 1]
```diff
@@ -732,3 +732,13 @@ evdev:input:b0003v172Fp0047*
...
+ 
+#########################################
+# Positivo-Vaio
+#########################################
+# Vaio FE14
+evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE41F11*:pvr*
+ EVDEV_ABS_00=::28
+ EVDEV_ABS_01=::27
+ EVDEV_ABS_35=::28
+ EVDEV_ABS_36=::27
```
Comment (whot) [inline hwdb.d/60-evdev.hwdb]: This file is ordered alphabetically by vendor, can you move this block up please to where it belongs, thanks

File: hwdb.d/60-evdev.hwdb [hunk 2]
```diff
@@ -732,3 +732,13 @@ evdev:input:b0003v172Fp0047*
 evdev:input:b0003v172Fp0505*
  EVDEV_ABS_00=::160
  EVDEV_ABS_01=::160
+ 
+#########################################
+# Positivo-Vaio
+#########################################
+# Vaio FE14
+evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:bvn*:bvr*:bd*:svnPositivoBahia-VAIO:pnVJFE41F11*:pvr*
```
Comment (whot) [inline hwdb.d/60-evdev.hwdb]:
```suggestion
evdev:name:SYNA3602:00 0911:5288 Touchpad:dmi:*svnPositivoBahia-VAIO:pnVJFE41F11*
```
No need to specify all the others with globs.

**Review by whot — CHANGES_REQUESTED** [review]

File: hwdb.d/60-evdev.hwdb (hunk 2 above)
Comment (davijosw) [reply hwdb.d/60-evdev.hwdb]: Hey, I made a pull request on the hwdb keyboard file and this particular line was pretty much the same (with globs). Do I have to make a new pull request to correct this aspect? Or the keyboard and the evdev are differents in the system reading aspect?

Comment (whot) [reply hwdb.d/60-evdev.hwdb]: hwdb files are all parsed by the same parser, if that's what you're referring to. Whether you specify all sub-components is just convention, looks like `60-keyboard.hwdb` primarily uses the long forma, `60-evdev.hwbd` primarily uses the shortened format. I think it's better to use the short version here for consistency (plus: it's less visual detritus).

**Comment by whot** [comment]
merged (with fixed commit message), thanks!

File: hwdb.d/60-evdev.hwdb (hunk 2 above)
Comment (davijosw) [reply hwdb.d/60-evdev.hwdb]: thanks!

**Pull request merged**